pkg/asset/quota/quota.go

```diff
@@ -157,24 +157,6 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to create a new PISession")
 		}
-
-		transitGatewayEnabled := configpowervs.TransitGatewayEnabledZone(ic.Config.Platform.PowerVS.Zone)
```

Comment:
@miyamotoh has a patch to check the datacenter so this will be reincluded somewhat once that merges

Reply:
That's good.  From the sound of your OP, it seems like this is advisory and a change is not needed.  Correct?

Reply:
correct!